recipes/poco/all/patches/1.13.0-0002-mysql-include.patch

```diff
@@ -0,0 +1,143 @@
+diff --git a/Data/MySQL/include/Poco/Data/MySQL/Binder.h b/Data/MySQL/include/Poco/Data/MySQL/Binder.h
```

Comment:
I'll check what could be done about this patch. Or Poco is including wrongly, or MySQL recipe could use ["include/mysql"] too.

Reply:
https://github.com/conan-io/conan-center-index/pull/22126